configs/config_example.json

```diff
@@ -1,157 +1,26 @@
 {
     "common": {
-        "lib": ["sklearn", "daal4py"],
         "data-format": ["pandas"],
         "data-order": ["F"],
         "dtype": ["float64"]
     },
     "cases": [
         {
-            "algorithm": "distances",
```

Comment:
Reason to change example config?

Reply:
I will revise this config for use in regular CI. Most likely I'll do it on small datasets, but for all benchmarks, what do you think?

Reply:
revert this config